lib/fit-curve.js

```diff
@@ -19,23 +19,23 @@
         }
     }
 
-    /**
-     *  @preserve  JavaScript implementation of
-     *  Algorithm for Automatically Fitting Digitized Curves
-     *  by Philip J. Schneider
-     *  "Graphics Gems", Academic Press, 1990
-     *
-     *  The MIT License (MIT)
-     *
-     *  https://github.com/soswow/fit-curves
+    /**
+     *  @preserve  JavaScript implementation of
+     *  Algorithm for Automatically Fitting Digitized Curves
+     *  by Philip J. Schneider
+     *  "Graphics Gems", Academic Press, 1990
+     *
+     *  The MIT License (MIT)
+     *
+     *  https://github.com/soswow/fit-curves

```

Comment:
What happened with all these white line changes?

Reply:
Sorry, these didn't show up in my editor's diff. It seems to be a LF vs CRLF newline issue. My editor now detects fit-curve.js as CRLF while other files seem to be LF. Maybe this file had internally inconsistent newlines and my editor auto-unified it? I didn't touch it now because I didn't want to create more of a mess. If it's ok I can just convert the whole file to LF and commit that.

Reply:
> If it's ok I can just convert the whole file to LF and commit that.
Yes, please! Let's do this.

Reply:
Done.